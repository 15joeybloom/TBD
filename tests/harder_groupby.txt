select DName, avg(Grade) from student, enroll, dept
where SId = StudentId and MajorId = DId
group by DName
exit
