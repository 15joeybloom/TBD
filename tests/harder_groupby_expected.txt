
SQL> 
select DName, avg(Grade) from student, enroll, dept where SId = StudentId and MajorId = DId group by DName
   DName   avg(Grade)
--------------
 compsci    69
   drama    83
    math    88

SQL> 
exit
